client/app/competitor/page.tsx

```diff
@@ -12,6 +13,37 @@ import {
 } from '@/components/ui/accordion';
 import { ScrollArea } from '@/components/ui/scroll-area';
 import { Separator } from '@/components/ui/separator';
+import Leaderboard from '../leaderboard/page';
+
+const ObserveLeaderboardTab = () => {
```

Comment:
Nit: this name is not descriptive.  Something like `TabContent` would be more descriptive.